scripts/run_trade_bot.sh

```diff
@@ -132,28 +132,99 @@ do
 
   for (( pair_index=0; pair_index<$token_pair_config_array_length; pair_index++ ))
   do
-    token_pair=$( echo "$token_pair_config_array" | jq -r ".[$pair_index].pair | sort_by(.denom)" )
+    token_pair=$( echo "$token_pair_config_array" | jq -r ".[$pair_index].pair" )
```

Comment:
The removal of the token sorting changes the behavior of the "rebalancing" (deposit and withdrawal) steps of the script. This is because the REST API endpoint `/neutron/dex/user/deposits/$address` returns the user's deposits in terms of ordered `token0` and `token1`. If the pair config was written with the reverse order then the deposits don't match as expected and fewer deposits occur.

I'd like the PR to add just a short description that mentions that the PAIR_CONFIG token order must be written in the correct order. 

I've found the logic to add a fix to make the script "token order agnostic" without flipping the tokens order or the price values, but I'd like to add it as a separate PR because it will touch many lines and make this PR's diff difficult to read.

Reply:
noting that the token order issue has been fixed and merged in
- https://github.com/neutron-org/dex-trading-bot/pull/9